src/main/java/cs361/battleships/models/Board.java

```diff
@@ -50,22 +48,7 @@ else if (ship.getKind().equals("DESTROYER")){
 		if (placedShip.getOccupiedSquares().stream().anyMatch(s -> s.isOutOfBounds())) {
 			return false;
 		}
-		if (ship.getKind().equals("MINESWEEPER")){
-			Minesweeper tempShip = new Minesweeper();
-			tempShip.place(y, x, isVertical);
-			ships.add(tempShip);
-		}
-		else if (ship.getKind().equals("DESTROYER")){
-			Destroyer tempShip = new Destroyer();
-			tempShip.place(y, x, isVertical);
-			ships.add(tempShip);
-		}
-		else {
-			Battleship tempShip = new Battleship();
-			tempShip.place(y, x, isVertical);
-			ships.add(tempShip);
-		}
-//		ships.add(placedShip);
```

Comment:
were these lines removed because the ships have been implemented polymorphically?

Reply:
This was some leftover code from when I was trying the polymorphism and for some reason, it got transferred over so I got rid of it.